rust/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+# 0.11.3
+
+* Updates from mediasoup TypeScript `3.10.13..=3.11.8`.
```

Comment:
I don't understand these changes. Could you explain each of them please?

Reply:
This is to avoid copy-pasting changelog, like previously only Rust-specific changes are mentioned explicitly.